webui/react/src/components/ExperimentIcons.tsx

```diff
@@ -48,8 +53,10 @@ const ExperimentIcons: React.FC<Props> = ({
       case RunState.StoppingCompleted:
       case RunState.StoppingError:
       case RunState.StoppingKilled:
+      case CommandState.Terminating:
         return { color: 'cancel', name: 'spin-shadow', title: stateToLabel(state) };
       case RunState.Canceled:
+      case CommandState.Terminated:
         return { color: 'cancel', name: 'cancelled', title: 'Stopped' };
```

Comment:
to avoid future weirdness, can we add a `default:` case? I would use the cancelled/stopped icon

Reply:
That's a good idea, but using cancel icon might confuse users because state can be anything. That would be cool if we have an unknown icon like `(?)`, but looks like hew doesnt have the icon so instead, can we use `exclamation-circle` or `warning` for default case? what do you think?

Reply:
`exclamation-circle` makes sense to me